openapi/code/entity.go

```diff
@@ -256,6 +256,15 @@ func (e *Entity) IsAllRequiredFieldsPrimitive() bool {
 	return true
 }
 
+func (e *Entity) IsAllRequiredFieldsJsonUnserialisable() bool {
```

Comment:
If I understand this correctly, the check returns if any of the required fields are not part of the request body (i.e. in the URL path or query). If so, then the name here should reflect that. E.g. HasRequiredNonBodyField.